elastic/kibana
# [Monitoring] Clearer MB migration state on the overview page

Resolves #41725 

Instead of showing `0/1` (or something similar), let's simplify what we're showing by only showing something (a flag icon) if the user _should_ take action on the listing page for the stack product. Also, provide a tooltip that explains this more.

<img width="430" alt="Screen Shot 2019-07-24 at 10 18 29 AM" src="https://user-images.githubusercontent.com/56682/61801340-93876a80-adfc-11e9-88fe-6dc080f6a216.png">
<img width="550" alt="Screen Shot 2019-07-24 at 10 18 34 AM" src="https://user-images.githubusercontent.com/56682/61801341-93876a80-adfc-11e9-8e2f-844fd06fe15f.png">

## Testing

- Load stack monitoring
- Navigate/Ensure on the `/overview` page
- Add the setup mode flag (`inSetupMode:!t`) to the global state, so a valid url will look like: http://localhost:5601/app/monitoring#/overview?_g=(cluster_uuid:SckmahAJRiG_Z0R47B-J1w,inSetupMode:!t)
